packages/helm-charts/celo-fullnode/templates/NOTES.txt

```diff
@@ -0,0 +1,19 @@
+1. Get the application URL by running these commands:
```

Comment:
I like this file, very good idea

Reply:
It is created automatically with the chart skeleton. It is the info shown just after deploying or upgrading the chart, we can complete with some meaningful information.